### PR TITLE
Link reviewing to the API

### PR DESCRIPTION
**Pull request by Pinpickle**
Starting to feel the need for tests here ;)

This makes reviewing actually work. Currently PDFs aren't displayed but I think that will be fixed shortly when they're made world-readable.

No errors are displayed when something goes wrong, which is something we ought to fix.

@jaredkhan @varkor could you review?

### CODE REVIEW

File: src/application/Application.js [hunk 1]
```diff
@@ -1,84 +1,86 @@
...
-          </FormGroup>
-          <Button color="success" className="next-button">Next</Button>
-        </Form>
-      </footer>
-    </div>
-  );
-}
+            <Card block className="cv-link">
+              <CardText>
+                <a href="http://www.cl.cam.ac.uk/teaching/1516/Algorithms/2016-stajano-algs-students-handout.pdf" target="_blank">
```
Comment (jaredkhan) [inline src/application/Application.js]: This needs to be the CV from the application too

**Review by jaredkhan — CHANGES_REQUESTED** [review]
Looks good! Just need to change that one link and maybe think about better feedback when you click the next button. e.g. The criteria dropdown should be set back to default values and maybe we flash some message about success, I suppose we could do errors and success messages via the same mechanism. Nice job.

File: src/application/NoReviews.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
```
Comment (jaredkhan) [inline src/application/NoReviews.js:3]: Want an `import { Container } from 'reactstrap';` here

**Review by jaredkhan — CHANGES_REQUESTED** [review]

File: src/application/Application.js (hunk 1 above)
Comment (Pinpickle) [reply src/application/Application.js]: Good catch

File: src/application/NoReviews.js (hunk 2 above)
Comment (Pinpickle) [reply src/application/NoReviews.js:3]: How did this even build??

File: src/application/ApplicationPage.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+function ApplicationPage({ application, criteria, onSubmit, adminId }) {
+  if ((!application) || (!criteria)) {
+    return <p>Loading...</p>;
+  }
+
+  return <Application
+    application={application}
+    criteria={criteria}
+    key={application.id}
```
Comment (Pinpickle) [inline src/application/ApplicationPage.js:15]: @jaredkhan this is what makes the dropdowns reset

**Review by jaredkhan — APPROVED** [review]
Looks great!

**Pull request merged**